src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java

```diff
@@ -74,6 +80,19 @@ public boolean includeNs() {
     }
 
     /**
+     * Does this tool emit information for uncovered loci? Tools that do should override to return {@code true}.
+     *
+     * NOTE:  Typically, this should only be used when intervals are specified.
+     * NOTE:  If MappedReadFilter is removed, then emitting empty loci will fail.
+     *
+     * @return {@code true} if this tool requires uncovered loci information to be emitted, {@code false} otherwise
+     */
+    public boolean emitEmptyLoci() {
```

Comment:
Recommend keeping this as a simple boolean flag -- adding a secondary layer of intervals seems overly complex, and it's unclear that there would be a use case for it.